### PR TITLE
feat: add loadbalancing to table.

### PR DESCRIPTION
**Pull request by tplive**
Looks like Data traffic is the new table, but that's just because I copied it and made changes for Loadbalancing.. :)

### CODE REVIEW

File: prices.html [hunk 1]
```diff
@@ -305,21 +309,37 @@
     </div>
 
     <div class="accs-price-table-wrapper">
-        <table id="table-network-traffic" class="accs-price-table">
+        <table id="table-loadbalancing" class="accs-price-table">
```
Comment (bakksjo) [inline prices.html]: I don't think LB should be a separate table. I would have added it to the Network Services table.

**Review by bakksjo — CHANGES_REQUESTED** [review]

File: prices.html (hunk 1 above)
Comment (tplive) [reply prices.html]: Updated as suggested.

File: prices.html [hunk 2]
```diff
@@ -303,23 +307,22 @@
...
-                </tr>
-            </thead>
-            <tbody>
-                <tr class="storage-row"><td class="storage">Ingress traffic (RX)</td><td class="gb-month"></td><td class="free">100 GB</td></tr>
-                <tr class="storage-row"><td class="storage">Egress traffic (TX)</td>  <td class="gb-month"></td> <td class="free">100 GB</td></tr>
-            </tbody>
-        </table>
-        <p>&nbsp;</p>
-        </div>
+        <div class="accs-price-table-wrapper">
```
Comment (bakksjo) [inline prices.html]: I don't really approve of this indent change; why is this div not aligned with the previous?

**Review by bakksjo — APPROVED** [review]

File: prices.html (hunk 2 above)
Comment (tplive) [reply prices.html]: Fixed indentation.

**Pull request merged**